lib/web/apiserver.go

```diff
@@ -531,16 +540,19 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*APIHandler, error) {
 
 	// Create application specific handler. This handler handles sessions and
 	// forwarding for application access.
-	appHandler, err := app.NewHandler(cfg.Context, &app.HandlerConfig{
-		Clock:         h.clock,
-		AuthClient:    cfg.ProxyClient,
-		AccessPoint:   cfg.AccessPoint,
-		ProxyClient:   cfg.Proxy,
-		CipherSuites:  cfg.CipherSuites,
-		WebPublicAddr: resp.SSH.PublicAddr,
-	})
-	if err != nil {
-		return nil, trace.Wrap(err)
+	var appHandler *app.Handler
+	if !cfg.MinimalReverseTunnelRoutesOnly {
```

Comment:
nit: I'd rather test for the positive here too (ie, handle minimal tunnels inside the if), but this is minor compared to the one above.

Reply:
Ping? Not addressed?

Reply:
Sorry, I should have said something earlier. I didn't change this one because there's nothing to do in the positive case.